next/test/validation/object.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it } from '@jest/globals'
+import { createHeadlessForm } from '../../src'
+
+describe('object schema validation', () => {
+  it('returns an error if the value is not an object', () => {
+    const schema = { properties: { address: { type: 'object', properties: { street: { type: 'string' } } } } }
+    const form = createHeadlessForm(schema)
+
+    expect(form.handleValidation({})).not.toHaveProperty('formErrors')
+    expect(form.handleValidation({ address: {} })).not.toHaveProperty('formErrors')
+    expect(form.handleValidation({ address: 'not an object' })).toMatchObject({
+      formErrors: { '.address': 'should be object' },
```

Comment:
bug/question: Why does the path has a `.` prefix. This is different from v0 and would break the internal code.